README.Rmd

```diff
@@ -50,13 +51,26 @@ options for a variety of operating systems:
 
 ### R-universe
 
-[R-universe](https://rpolars.r-universe.dev/rpolars#install) provides
+[R-universe](https://rpolars.r-universe.dev/polars#install) provides
 pre-compiled **polars** binaries for Windows and MacOS (x86_64), with source
-builds for other platforms. Please see the GitHub release option below for
-binary install options on Linux.
+builds for other platforms.
 
 ```r
-install.packages("polars", repos = "https://rpolars.r-universe.dev")
+install.packages(
+  "polars", 
+  repos = "https://rpolars.r-universe.dev"
+)
```

Comment:
(Style nit, personal)  I really do not like that indentation.  The original single line 59 is so much easier to read, grasp and copy than new four lines 59 to 62.